plenum/server/consensus/ordering_service_msg_validator.py

```diff
@@ -122,6 +122,14 @@ def _validate_3pc(self, msg) -> Tuple[int, Optional[str]]:
         if pp_seq_no is not None and pp_seq_no > self._data.high_watermark:
             return STASH_WATERMARKS, OUTSIDE_WATERMARKS
 
+        # Allow re-order already ordered if we are re-ordering in new view
+        # (below prepared certificate from the previous view).
```

Comment:
Do we have unit tests for the new validation logic? (in `test_ordering_msg_validator.py `)